src/domain/entities/alert/Alert.ts

```diff
@@ -0,0 +1,9 @@
+export type Alert = {
+    trackedEntity: string | undefined;
```

Comment:
domain entity should never have dhis verbiage - this is a clear code smell. Should this be in a data repository?

Reply:
Totally agree, perhaps this could be fixed by a better definition of the entity with better names in each property instead of referring to dhis2 names